### PR TITLE
Change "Enter" submit logic

### PR DESCRIPTION
**Pull request by Pelps12**
This PR enables submit on "Enter" in the textbox and newlines with (Shift + Enter). Following common chat box convention.

### CODE REVIEW

File: apps/web/src/components/Main.tsx [hunk 1]
```diff
@@ -388,7 +388,7 @@ export default function Main({ sidebarOpen }: { sidebarOpen: boolean }) {
                 autoFocus: true,
                 onChange: (e) => setValue(e.target.value),
                 onKeyDown: (e) => {
-                  if (e.key === "Enter" && (e.ctrlKey || e.metaKey)) {
+                  if (e.key === "Enter" && !e.shiftKey) {
                     onSend();
```
Comment (Dhravya) [inline apps/web/src/components/Main.tsx:394]: can you please add a `e.preventDefault()` here? 
Hitting enter will also create a newline (and run the onsend() fn)

preventDefault prevents this

File: apps/web/src/components/Main.tsx [hunk 2]
```diff
@@ -507,7 +507,7 @@ export function Chat({
               autoFocus: true,
               onChange: onValueChange,
               onKeyDown: (e) => {
-                if (e.key === "Enter" && (e.ctrlKey || e.metaKey)) {
+                if (e.key === "Enter" && !e.shiftKey) {
```
Comment (Dhravya) [inline apps/web/src/components/Main.tsx:512]: e.preventDefault()

**Review by Dhravya — CHANGES_REQUESTED** [review]

File: apps/web/src/components/Main.tsx (hunk 2 above)
Comment (Pelps12) [reply apps/web/src/components/Main.tsx:512]: Added

**Review by Dhravya — APPROVED** [review]

**Comment by Dhravya** [comment]
Thanks!

**Pull request merged**